### PR TITLE
Fix typos on comments

### PR DESCRIPTION
**Pull request by altmas5**
I removed consecutive occurrences of 'the'
`s/the the /the /g`

### CODE REVIEW

**Comment by altmas5** [comment]
> Hi, thanks for your contribution!
> If you haven't already done so, could you please make sure you sign our CLA (https://jitsi.org/icla for individuals and https://jitsi.org/ccla for corporations)? We would unfortunately be unable to merge your patch unless we have that piece :(.

Hi @jitsi-jenkins I signed!

File: modules/statistics/CallStats.js [hunk 1]
```diff
@@ -69,7 +69,7 @@ let _fabrics;
 
 /**
  * An instance of this class is a wrapper for the CallStats API fabric. A fabric
- * reports one peer connection the the CallStats backend and is allocated with
+ * reports one peer connection the CallStats backend and is allocated with
```
Comment (jallamsetty1) [inline modules/statistics/CallStats.js:72]: should be `reports one peer connection to the callstats backend.`

File: modules/transcription/transcriber.js [hunk 2]
```diff
@@ -85,7 +85,7 @@ Transcriber.prototype.stop = function stop(callback) {
     console.log('stopping recording and sending audio files');
     this.audioRecorder.stop();
 
-    // and send all recorded audio the the transcription service
+    // and send all recorded audio the transcription service
```
Comment (jallamsetty1) [inline modules/transcription/transcriber.js:88]: should be - `and send all recorded audio to the transcription service.`

**Review by jallamsetty1 — COMMENTED** [review]
@altmas5 Thanks for the PR, can you pls make the 2 changes suggested ?

File: modules/transcription/transcriber.js (hunk 2 above)
Comment (altmas5) [reply modules/transcription/transcriber.js]: Hi @jallamsetty1 I just did

**Review by jallamsetty1 — APPROVED** [review]

**Pull request merged**